src/io/file_descriptor.cr

```diff
@@ -133,6 +133,60 @@ class IO::FileDescriptor < IO
     value
   end
 
+  # Flushes all data written to this File to the disk device so that
```

Comment:
Maybe change File to File Descriptor? Same in other places if it happens.

Reply:
Done. Thank you @asterite